spec.bs

```diff
@@ -1374,8 +1374,13 @@ an unauthorized payment using Secure Payment Confirmation credentials
     assertion for the [=Relying Party=].
 * If the [=Relying Party=] is not expecting a transaction, it will reject the
     assertion.
-* If the [=Relying Party=] is expecting a transaction, it will detect an
-    unfamiliar `challenge` and reject the assertion.
+* If the [=Relying Party=] is expecting a transaction, multiple fields it will
+    detect at least one of the following and reject the assertion:
```

Comment:
> multiple fields it will detect at least one of the following

This sentence no grammar?

Reply:
o.O

Fixed, thanks. Strange... vim buffer failure?! (Probably not. Probably just Stephen failure).